EasyPost/Models/API/Address.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using EasyPost._base;
+using EasyPost.Utilities.Annotations;
+using Newtonsoft.Json;
+using RestSharp;
+
+namespace EasyPost.Models.API
+{
+    public class Address : EasyPostObject
+    {
+        #region JSON Properties
+
+        [JsonProperty("carrier_facility")]
+        public string carrier_facility { get; set; }
+
+        [JsonProperty("city")]
+        public string city { get; set; }
+        [JsonProperty("company")]
+        public string company { get; set; }
+        [JsonProperty("country")]
+        public string country { get; set; }
+        [JsonProperty("email")]
+        public string email { get; set; }
+        [JsonProperty("error")]
+        public string error { get; set; }
+        [JsonProperty("federal_tax_id")]
+        public string federal_tax_id { get; set; }
+        [JsonProperty("message")]
+        public string message { get; set; }
+        [JsonProperty("mode")]
+        public new string mode { get; set; }
+        [JsonProperty("name")]
+        public string name { get; set; }
+        [JsonProperty("phone")]
+        public string phone { get; set; }
+        [JsonProperty("residential")]
+        public bool? residential { get; set; }
+        [JsonProperty("state")]
+        public string state { get; set; }
+        [JsonProperty("state_tax_id")]
+        public string state_tax_id { get; set; }
+        [JsonProperty("street1")]
+        public string street1 { get; set; }
+        [JsonProperty("street2")]
+        public string street2 { get; set; }
+        [JsonProperty("verifications")]
+        public Verifications verifications { get; set; }
+        [JsonProperty("verify")]
+        public List<string> verify { get; set; }
+        [JsonProperty("verify_strict")]
+        public List<string> verify_strict { get; set; }
+        [JsonProperty("zip")]
+        public string zip { get; set; }
```

Comment:
Thought: I'm torn here. On the one hand sorting things like this is great because it's easy to find things and maintain the list. On the other, it breaks the logical ordering of important things like addresses where it's typically street, state, zip, etc. 

I suppose it's probably not a deal **in-code** and that sort of thing is only important at the presentation layer which we aren't concerned with here. What do you think?

Reply:
I prefer alphabetically, since it will inevitably get unsorted if we try to maintain a manual order.

Reply:
Agreed, my brain will hurt but since it's internal code I think alphabetically ordering is the best option here.